Tasks/GitHubPRComment/githubComment/index.ts

```diff
@@ -3,68 +3,131 @@ import gitClient = require('@octokit/rest');
 import path = require('path');
 import fs = require('fs');
 
-const clientWithAuth = new gitClient({
-    auth: "token "+ taskLibrary.getInput('userToken'),
-    userAgent: 'octokit/rest.js v1.2.3',
-});
+//We use HTML comments to hide the flagComment in the comment.
+const flagComment = "<!-- Commented using GitHubPRComment. -->";
```

Comment:
I think the comment after the `//` should start with a space.
the flagComment, i think we could add a clarification saying something `DO NOT DELETE THESE COMMENT` or something like that.

Reply:
Something like this?
<img width="565" alt="image" src="https://user-images.githubusercontent.com/64815358/180492582-a52f11b6-eeaa-4cf0-807f-16c88cd2a793.png">

Reply:
THESE -> THIS